var/spack/repos/builtin/packages/py-black/package.py

```diff
@@ -25,29 +26,30 @@ class PyBlack(PythonPackage):
     # This is the last v21 release, and it's needed to format for Python 2.7
     version("21.12b0", sha256="77b80f693a569e2e527958459634f18df9b0ba2625ba4e0c2d5da5be42e6f2b3")
 
-    variant("d", default=False, description="enable blackd HTTP server")
     variant("colorama", default=False, description="enable colorama support")
     variant("uvloop", default=False, description="enable uvloop support")
+    variant("d", default=False, description="enable blackd HTTP server")
```

Comment:
Curious with the motivation for the variant ordering.  Why move `d` here?

Reply:
I try to keep the variant and dependency order the same as documented in `pyproject.toml`/`setup.py`/`setup.cfg`. The developers moved things around, so I did too.